.moai/specs/SPEC-WT-DOC-001/research.md

```
@@ -0,0 +1,195 @@
+- 또는 `git fetch origin && git log origin/main --grep "SPEC-LOOP-TERM-001"`
+
+---
+
+## 1. 출처 (Anthropic 공식 자료)
+
+### 1.1 Verbatim 인용
+
+Anthropic blog "Multi-Agent Coordination Patterns" (Shared State 절):
+
+> "Reactive loops are a behavioral problem requiring first-class termination conditions."
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)